.github/workflows/build.yml

```
@@ -0,0 +1,68 @@
+name: Build benchmark docs
+
+on: 
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    #- name: Check out data repository
+    #  uses: actions/checkout@v2
+    #  with:
+    #    repository: //<insert your repository here>
+    #    token: ${{ secrets.GIT_TOKEN }}
+
+    - uses: actions/checkout@v2
+
+    - name: Set up Python 3.10
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.9'
```

Probably a small typo but should this be:

```suggestion
    - name: Set up Python 3.9
      uses: actions/setup-python@v2
      with:
        python-version: '3.9'
```